src/components/Users.jsx

```diff
@@ -0,0 +1,5 @@
+const Users =  () => {
```

Comment:
As we're not currently using this element, it would be good not to include in this PR
It's best in PRs to only include code that is needed to implement the ticket/task